src/ngx_http_lua_headers.c

```diff
@@ -145,10 +155,27 @@ ngx_http_lua_ngx_req_raw_header(lua_State *L)
 
     dd("size: %d", (int) size);
 
+#if defined(nginx_version) && nginx_version >= 1011011
+    busy_bufs = ngx_array_create(r->pool, hc->nbusy + 1, sizeof(ngx_buf_t *));
```

Comment:
Does not appears to me that the `+ 1` here is necessary when creating the array.